container/sleepysh

```diff
@@ -1,3 +1,11 @@
 #!/bin/sh
+
+# handle the case of sshuttle
+if [[ "$@" == *"sys.stdin.read"* ]]; then
+    shift
+    sh -c "$@"
+fi
```

Comment:
To confirm, this is:

if the arguments to sleepysh contain "sys.stdin.read", then shift the arguments by 1 and run the rest as a command in `sh` ?

Reply:
Exactly that!